watchtower/wtclient/client.go

```diff
@@ -1659,7 +1659,7 @@ func (c *TowerClient) LookupTower(pubKey *btcec.PublicKey,
 
 // Stats returns the in-memory statistics of the client since startup.
 func (c *TowerClient) Stats() ClientStats {
-	return c.stats.Copy()
+	return c.stats.ClientStats
```

Comment:
Do we still need to hold the lock here?

Reply:
nice catch - added a `getStatsCopy` method to the `clientStats` which then grabs the lock 👍